leader-election-impl/src/test/java/com/palantir/leader/proxy/AwaitingLeadershipProxyTest.java

```diff
@@ -103,42 +96,6 @@ public void shouldAllowObjectMethodsWhenNotLeading() {
         assertThat(proxy.toString()).startsWith("com.palantir.leader.proxy.AwaitingLeadershipProxy@");
     }
 
-    @Test
-    public void shouldNotRedirectToItself() {
```

Comment:
these tests make no sense, if anything, this should be tested at the `PaxosLeaderElectionService` level. 

`LeaderElectionService` provides the `currentToken`, and you should be able to tell whether or not you're the current leader or not. Either way, I'm struggling to find the value in these tests or the `getSuspectedLeaderInMemory` method whatsoever. So they're gone. 🤷‍♂